z.psm1

```diff
@@ -7,6 +7,16 @@
 }
 $cdHistory = Join-Path -Path $safehome -ChildPath '\.cdHistory'
 
+# The user can set $Z_UsePushLocation to $true to configure z to use Push-location, which allows to use popd/Pop-Location to go back to the previous location.
+# Setting or $Z_UsePushLocation to $false will use Set-Location instead.
+
+if ($Z_UsePushLocation -eq $null)
```

Comment:
Best practice is to have $null on the left hand side of the condition.
https://learn.microsoft.com/en-us/powershell/scripting/learn/deep-dives/everything-about-null?view=powershell-7.3#checking-for-null

```suggestion
if ($null -eq $Z_UsePushLocation)
```